build_library/build_image_util.sh

```diff
@@ -452,8 +452,18 @@ EOF
     sudo lbzip2 -9 "${root_fs_dir}"/usr/share/licenses/licenses.json
     # Copy all needed licenses to a "common" subdirectory and compress them
     local license_list # define before assignment because it would mask any error
-    license_list="$(jq -r '.[] | "/var/gentoo/repos/gentoo/licenses/\(.licenses | .[])"' "${json_input}" | sort | uniq)"
-    sudo cp ${license_list} "${root_fs_dir}"/usr/share/licenses/common/
+    license_list="$(jq -r '.[] | "\(.licenses | .[])"' "${json_input}" | sort | uniq)"
+    for license_file in ${license_list}; do
+      if [ -f "/var/gentoo/repos/gentoo/licenses/${license_file}" ]; then
```

Comment:
I think a nested loop that iterates over possible directories here would make it a bit more readable (less repeated code, in particular less repeated routes):

```
LICENSE_DIRS=(
"/var/gentoo/repos/gentoo/licenses",
"/mnt/host/source/src/third_party/portage-stable/licenses",
"/mnt/host/source/src/third_party/coreos-overlay/licenses"
)
for license_file in ${license_list}; do
  for license_dir in ${LICENSE_DIRS}; do
    if [ -f "$license_dir/$license_file" ]; then
      cp "$license_dir/$license_file" "${root_fs_dir}"/usr/share/licenses/common/
    fi
  done
done
```

Reply:
Yes, would work when I add a dummy entry as end of the list to print the warning when no file was found

Reply:
Updated it (Alternatives would be "break N" or some state variable)